textdb/textdb-angular-gui/app/services/mock-data.ts

```diff
@@ -389,9 +389,33 @@ let wordCount = {
       }
     },
     attributes : {
-        "operatorType": "WordCount",
-        "attribute": "attr1",
-	"luceneAnalyzer": "standard",
+      "operatorType": "WordCount",
+      "attributes": [],
+      "luceneAnalyzer": "standard",
+    }
+  }
+}
+
+let comparison = {
```

Comment:
Why this `comparison` operator is here?  It shouldn't be in this autocomplete PR